src/format.ts

```diff
@@ -55,8 +56,8 @@ function queryToItem(
     request: {
       method: "POST",
       header: [
-        ...(authorization
-          ? [{ key: "Authorization", value: authorization }]
+        ...((authorization && authorizationHeader)
+          ? [{ key: authorizationHeader.trim(), value: authorization.trim() }]
```

Comment:
This would set our auth key as the value for the generated requests right? We don't want that. Better to use an env variable and the person who downloads the collection can add it to their Postman. We can also have the generated README reflect this. Something like `._storefront_access_token` which is how we currently do it in the storefront-api-learning-kit.

Reply:
For now, this PR would only set the blank values for the custom headers set in the postman collection. 

My 2 cents: I think we can mention this in the README to fill it up for now. We shall look at the enhancement of doing this once we complete the MVP. What do you think?